Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -297,7 +294,7 @@ private void SendEntitySpeak(EntityUid source, string originalMessage, bool hide
 
         SendInVoiceRange(ChatChannel.Local, message, wrappedMessage, source, hideChat, hideGlobalGhostChat);
 
-        var ev = new EntitySpokeEvent(source, message, channel, null);
+        var ev = new EntitySpokeEvent(source, message, null, null);
```

Comment:
a couple subscribers (headset/intrinsicradio) use the channel field on `EntitySpokeEvent`, so I suppose now the channel field only goes through on the whispered message, which seems kind of odd to me? there should probably be comments about this behavior at least

Reply:
I can't say I understand these events really - so maybe I've missed some consequence of this change.
But previously this would have never been called with anything but null. because if the radio had a channel then it returned early here: https://github.com/space-wizards/space-station-14/pull/12834/files#diff-0eb5df8d137cd4711bbde50acf9057a12a5c1efbb03badecabbb635af4d9fcb5L274

Reply:
yea makes sense, the flow here is really weird

Reply:
@mirrorcult Did you want me to take a stab at throwing some comments somewhere about this? I'm not really clear sorry.